README.md

```diff
@@ -34,22 +32,62 @@ vars:
     linkedin_schema: your_schema_name
 
 ```
-
 For additional configurations for the source models, please visit the [LinkedIn Ad Analytics source package](https://github.com/fivetran/dbt_linkedin_source).
 
-## Contributions
+### Switching to Local Currency
+Additionally, the package allows you to select whether you want to add in costs in USD or the local currency of the ad. By default, the package uses USD. If you would like to have costs in the local currency, add the following variable to your `dbt_project.yml` file:
+
+```yml
+# dbt_project.yml
 
+...
+config-version: 2
+
+vars:
+    linkedin__use_local_currency: True
+```
+
+### Passing Through Additional Metrics
+By default, this package will select `clicks`, `impressions`, and `costs` from the source `ad_analytics_by_creative` table to store into the `linkedin__ad_adapter` model. If you would like to pass through additional metrics to the ad adapter model, add the following configuration to your `dbt_project.yml` file:
+
+```yml
+# dbt_project.yml
+
+...
+vars:
+    linkedin__passthrough_metrics: ['the', 'list', 'of', 'metric', 'columns', 'to', 'include'] # from LINKEDIN_ADS.AD_ANALYTICS_BY_CREATIVE
+```
+
+### Changing the Build Schema
+By default this package will build the LinkedIn Ad Analytics models within a schema titled (<target_schema> + `_linkedin`) in your target database. If this is not where you would like your modeled LinkedIn data to be written to, add the following configuration to your `dbt_project.yml` file:
+
+```yml
+# dbt_project.yml
+
+...
+models:
+    linkedin:
+      +schema: my_new_schema_name # leave blank for just the target_schema
```

Comment:
I think it was helpful in a previous transform package you included the source model custom schema as well and I think it helped add value to this section. 
```suggestion
models:
    linkedin:
      +schema: my_new_schema_name # leave blank for just the target_schema
    linkedin_source:
      +schema: my_new_schema_name # leave blank for just the target_schema
```

Reply:
ah yeah good call